sirepo/job_supervisor.py

```diff
@@ -295,6 +303,58 @@ def __db_write(self):
         sirepo.util.json_dump(self.db, path=self.__db_file(self.db.computeJid))
         return self
 
+    @classmethod
+    def _get_running_jobs(cls, want_adm=False):
+        def _get_jobs():
+            o = []
+            for i in filter(
+                    lambda x: x.db.status == job.RUNNING,
+                    cls.instances.values(),
+            ):
+                d = i.db
+                s = d.computeJobStart
+                l = d.lastUpdateTime
+                r = [
+                    d.simulationType,
+                    d.simulationId,
+                    _strftime(s),
+                    _strftime(l),
+                    round((l - s) / 60.0, 2),
```

Comment:
Why not use a format here, even an f string? `f'{d.lastUpdateTime - d.computeJobStart:.1f}'` An alternative might be `h:m:s` format so you wouldn't have to explain in the header. Jobs might run for hours.